docs/src/documentation/03-components/05-input/inputfield/01-guidelines.mdx

```diff
@@ -52,15 +52,11 @@ making it easier for users to complete the field.
 
 <!-- eslint-disable -->
 
-#### Help message
+#### Error and Help messages
```

Comment:
In InputFile the order is opposite, "Help and Error messages", so I recommend being consistent.